env.sample.sh

```diff
@@ -49,11 +33,7 @@ else
     export ENV=$1
     export APP_NAME="veda-data-pipelines"
 
-    export VPC_ID=$vpcId
-    export SECURITY_GROUP_ID=$sgId
-    export SECRET_NAME=$pgSecret
-
-    export STAC_INGESTOR_API_URL=$stacIngestorUrl
+        export STAC_INGESTOR_URL=$stacIngestorUrl
```

Comment:
`STAC_INGESTOR_URL`  -> `STAC_INGESTOR_API_URL` ?

Reply:
ugh I saw that one and thought I had fixed it. Thank you !